bin/query-cloudwatch

```diff
@@ -64,13 +64,21 @@ class QueryCloudwatch
     end
   end
 
+  def fetch(&block)
+    cloudwatch_client.fetch(
+      query: config.query,
+      from: config.from,
+      to: config.to,
+    )
```

Comment:
I think we need to pass the block in or it wont go anywhere
```suggestion
      &block,
    )
```

Reply:
Whoops, yep, forgot that.

Reply:
Added in f01fce531.